playground/AzurePostgres/AzurePostgres.Api/AzurePostgres.Api.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Let's not make a whole new playground app for this.

I've updated https://github.com/dotnet/aspire/commit/c8315db2fa540c05a84060b23b21fc54389b96dd to allow both Azure and a Postgres container. You should be able to use that in your testing.